### DISCUSSION TITLE
Updated default inputs for Reopt API V3

### DISCUSSION THREAD

**Discussion by d-rao** [NREL/REopt-API-Analysis#135] [V3 Transition]
I was wondering if the team has updated the default values for the input fields for V3. If so, can you direct me to them? The v2_v3_inputs_map.csv doesn't seem to have the latest defaults. Thanks!

**Comment by adfarth** [comment]
The easiest places to find the latest default values are: [REopt.jl documentation](https://nrel.github.io/REopt.jl/dev/reopt/inputs/#ElectricTariff) (default values are listed next to each input name), [REopt_API models.py file](https://github.com/NREL/REopt_API/blob/master/reoptjl/models.py), or https://developer.nrel.gov/api/reopt/stable/help?API_KEY=DEMO_KEY. 

The latest defaults should also be shown in the [REopt web tool](https://reopt.nrel.gov/).
